### PR TITLE
LPS-94654 export-import-test: Create ExportImportMultiLanguageSearchTest

### PR DESCRIPTION
**Pull request by brandizzi**

### CODE REVIEW

**Comment by foshiro-zz** [comment]
ci:test:sf

**Comment by foshiro-zz** [comment]
ci:test:search

File: modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/search/test/ExportImportMultiLanguageSearchTest.java [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+import com.liferay.portal.kernel.test.rule.DeleteAfterTestRun;
+import com.liferay.portal.kernel.test.rule.SynchronousDestinationTestRule;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.util.LocaleThreadLocal;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.search.test.util.FieldValuesAssert;
+import com.liferay.portal.search.test.util.IndexerFixture;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerTestRule;
```
Comment (foshiro-zz) [inline modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/search/test/ExportImportMultiLanguageSearchTest.java]: Please, change to PermissionCheckerMethodTestRule

File: modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/search/test/ExportImportMultiLanguageSearchTest.java [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+ */
+@RunWith(Arquillian.class)
+public class ExportImportMultiLanguageSearchTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerTestRule.INSTANCE,
```
Comment (foshiro-zz) [inline modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/search/test/ExportImportMultiLanguageSearchTest.java]: Please, change to PermissionCheckerMethodTestRule

**Review by foshiro-zz — CHANGES_REQUESTED** [review]
@vagner-castro Code and tests looks good to me. Just have to change PermissionCheckerTestRule to PermissionCheckerMethodTestRule

**Comment by vagnerbc** [comment]
@foshiro Changes updated

**Comment by foshiro-zz** [comment]
ci:test:sf

**Comment by foshiro-zz** [comment]
ci:test:search

**Review by foshiro-zz — APPROVED** [review]

**Comment by foshiro-zz** [comment]
Submitted https://github.com/brandizzi/liferay-portal/pull/779

**Pull request closed without merging**